src/_cffi_src/openssl/callbacks.py

```diff
@@ -16,45 +14,24 @@
 """
 
 TYPES = """
-static const long Cryptography_STATIC_CALLBACKS;
-
-/* crypto.h
- * CRYPTO_set_locking_callback
- * void (*cb)(int mode, int type, const char *file, int line)
- */
-extern "Python" void Cryptography_locking_cb(int, int, const char *, int);
-
-/* pem.h
- * int pem_password_cb(char *buf, int size, int rwflag, void *userdata);
- */
-extern "Python" int Cryptography_pem_password_cb(char *, int, int, void *);
-
-/* rand.h
- * int (*bytes)(unsigned char *buf, int num);
- * int (*status)(void);
- */
-extern "Python" int Cryptography_rand_bytes(unsigned char *, int);
-extern "Python" int Cryptography_rand_status(void);
+typedef struct {
+    char *password;
+    int length;
+    int called;
+    int error;
+    int maxsize;
+} CRYPTOGRAPHY_PASSWORD_CB_ST;
```

Comment:
ST as in struct.  If we make it longer then ` CRYPTOGRAPHY_PASSWORD_CB_ST *st = (CRYPTOGRAPHY_PASSWORD_CB_ST *)userdata;` will be an unpleasant line to wrap for the C code. 😀 Maybe `CRYPTOGRAPHY_PASSWORD_DATA`?